src/abi/tests.rs

```diff
@@ -797,7 +896,7 @@ contract Testing {
 
     assert_eq!(idl.instructions[1].name, "getNum_uint64_uint64");
     assert!(idl.instructions[1].docs.is_none());
-    assert!(idl.instructions[1].accounts.is_empty());
+    assert_eq!(idl.instructions[1].accounts.len(), 1);
```

Comment:
How about checking if we have the correct account?

Reply:
Fixed